docs/en/connector-v2/source/Iceberg.md

```diff
@@ -90,8 +90,10 @@ libfb303-xxx.jar
-| common-options           |         | no       | -                    | Source plugin common parameters, please refer to [Source Common Options](../source-common-options.md) for details.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| increment.scan-interval  | long    | no       | 2000                 | The interval of increment scan(mills)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| common-options           |         | no       | -                    | Source plugin common parameters, please refer to [Source Common Options](../source-common-options.md) for details.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| filter                   | String  | no       | -                    | The where clause to filter the iceberg data. It mustn't contain the table name. For example: There is an iceberg table contains an `id` column with the `int` type, now we want to filter the data that `id`>100. Then you can input `id > 100`.                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
why not use sql just like iceberg sink did? https://github.com/apache/seatunnel/blob/7adf523a71a9f41a439231171d348bf514abf2c1/seatunnel-connectors-v2/connector-iceberg/src/main/java/org/apache/seatunnel/connectors/seatunnel/iceberg/sink/IcebergSinkFactory.java#L75 cc @hailin0

Reply:
@Hisoka-X  The `custom_sql` is expecting to receive a DELETE DML in the iceberg sink config  https://github.com/apache/seatunnel/blob/7adf523a71a9f41a439231171d348bf514abf2c1/docs/en/connector-v2/sink/Iceberg.md?plain=1#L81. I think it would be weird and misleading to ask users to set this in the source configs.

Reply:
I mean source can use same style sql to filter data. E.g.
```
select key1 from table where id = 2.
```

Reply:
@Hisoka-X +1

Why not use sql query define columns filter and row filter.

e.g:
`
query = "select f1, f2, ... from your table where c1 = xxx and c2 = xxx or ...."
`

Reply:
@hailin0 @Hisoka-X PTAL again.